### PR TITLE
Fancy header area

### PR DESCRIPTION
**Pull request by joyheron**
Because I somehow can't keep tweaking things and playing around with things.

This adds a separate styling to the main page so that the section directly under the title page has a light purple gray background. It also gives the first paragraph a slightly larger font-size, and fixes the spacing for the figcaption.

These are mainly just tweaks to existing design and actually is based on the #24, so if this branch is merged, that one will also be merged with it (I probably should have made the changes in the same branch, but that change I find more important (because it improves the semantic HTML) and this one is purely a design thing which really has no priority).

### CODE REVIEW

File: lib/styles/index.scss [hunk 1]
```diff
@@ -78,6 +78,7 @@ dd {
 /* components */
 @import "../../views/components/breadcrumb/breadcrumb";
 @import "../../views/components/container/container";
+@import "../../views/components/header-area/header-area";
```
Comment (FND) [inline lib/styles/index.scss:82]: It appears you forgot to commit this file.

File: views/components/code_example/_code_example.scss [hunk 2]
```diff
@@ -35,8 +35,19 @@
...
 	}
+}
+figure {
+	margin: 5rem auto;
 
-	&--wide-margin {
-		margin: 5rem auto;
+	figcaption {
+		max-width: 40rem;
+		margin: 0.5rem auto;
```
Comment (FND) [inline views/components/code_example/_code_example.scss]: This looks a a little cramped to me on the landing page; a little more vertical spacing?

**Review by FND — APPROVED** [review]
This looks very good! Just a couple of observations; feel free to merge at will.

It does occur to me that the default link styling is more obviously garish here than before, so we should probably address that soon (not a blocker though).

**Comment by joyheron** [comment]
Screen Shot 2020-02-28 at 11 25 01

@FND what do you think?

**Comment by FND** [comment]
Looks pretty good!

For this particular purpose though, I'm wondering whether the caption is a bit too prominent: It was originally intended as a more of a footnote. But maybe(!) what confuses me is that the code blocks' border ends below rather than above the caption? I'm quite possibly overthinking it. IOW :shipit: .

**Comment by joyheron** [comment]
Screen Shot 2020-02-28 at 14 43 45
Do you think this looks better?

**Comment by FND** [comment]
I'd say so, yes - though again: IANAD 🤷‍♂ 

Let's go ahead and merge it. We can always decide later whether further tweaking is warranted (e.g. toning down the border color to be less in-your-facey; something like `#AAA` perhaps - but see above).

**Pull request merged**